docs/server/source/http-client-server-api.rst

```diff
@@ -154,7 +154,15 @@ Transactions
    is in a committed block.
 
    .. note::
-   
```

Comment:
I think a blank line might be necessary to keep Spinx happy.

Reply:
It rendered fine for me. Other notes here also have a blank line.

Reply:
ok